cabby/geo/visualize.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Library to support geographical visualization.'''
+
+import folium
+import geopandas as gpd
+from json import JSONDecoder, JSONDecodeError
+import json
```

Comment:
Let's just use the json import and refer to json.JSONDecoder and json.JSONDecodeError.

Reply:
I removed both imports as I am not using them in the code